D1673/P1673.md

```diff
@@ -7199,9 +7164,10 @@ T dot(ExecutionPolicy&& exec,
 `v2.static_extent(0)` equals `dynamic_extent`, then
 `v1.static_extent(0)` equals `v2.static_extent(0)`.
 
-[4]{.pnum} *Effects:* Let `N` be `v1.extent(0)`.
-If `N` is zero, returns `init`, else returns
-*GENERALIZED_SUM*(`plus<>()`, `init`, `v1[0]*v2[0]`, ..., `v1[N-1]*v2[N-1]`).
+[4]{.pnum} *Returns:*
+
+  * [4.1]{.pnum} `init` if `v1.extent(0) == 0`.
+  * [4.2]{.pnum} Otherwise, *GENERALIZED_SUM*(`plus<>()`, `init`, `v1[0]*v2[0]`, ..., `v1[N-1]*v2[N-1]`).
```

Comment:
Do we need to define `N` here?

```suggestion
  * [4.2]{.pnum} Otherwise, *GENERALIZED_SUM*(`plus<>()`, `init`, `v1[0]*v2[0]`, ..., `v1[N-1]*v2[N-1]`), where `N` is `v1.extent(0)`.
```

Reply:
Falcepalm!  I totally missed that `N` is re-used in the the `GENERALIZED_SUM`!